docs/content/posts/2023-11-13-release-1.17.2.md

```diff
@@ -0,0 +1,243 @@
+[JIRA](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12353260).
+ 
+We highly recommend all users upgrade to Flink 1.17.2.
+ 
+# Release Artifacts
+ 
+## Maven Dependencies
+ 
+```xml
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-java</artifactId>
+  <version>1.17.2</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-streaming-java</artifactId>
+  <version>1.17.2</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-clients</artifactId>
+  <version>1.17.2</version>
+</dependency>
+```
+ 
+## Binaries
+ 
+You can find the binaries on the updated [Downloads page]({{< relref "downloads" >}}).
+ 
+## Docker Images
+ 
+* [library/flink](https://hub.docker.com/_/flink/tags?page=1&name=1.17.2) (official images)
+* [apache/flink](https://hub.docker.com/r/apache/flink/tags?page=1&name=1.17.2) (ASF repository)
+ 
+## PyPi
+ 
+* [apache-flink==1.17.2](https://pypi.org/project/apache-flink/1.17.2/)
+ 
+
+        Release Notes - Flink - Version 1.17.2
+    
+<h2>        Sub-task
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-31774'>FLINK-31774</a>] -         Add document for delete and update statement
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-32316'>FLINK-32316</a>] -         Duplicated announceCombinedWatermark task maybe scheduled if jobmanager failover
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-32362'>FLINK-32362</a>] -         SourceAlignment announceCombinedWatermark period task maybe lost
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-32411'>FLINK-32411</a>] -         SourceCoordinator thread leaks when job recovers from checkpoint
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-32414'>FLINK-32414</a>] -         Watermark alignment will cause flink jobs to hang forever when any source subtask has no SourceSplit
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-32457'>FLINK-32457</a>] -         update current documentation of JSON_OBJECTAGG/JSON_ARRAYAGG to clarify the limitation
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-32458'>FLINK-32458</a>] -         support mixed use of JSON_OBJECTAGG &amp; JSON_ARRAYAGG with other aggregate functions
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-32496'>FLINK-32496</a>] -         Sources with idleness and alignment always wait for alignment when part of multiple sources is idle
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-32869'>FLINK-32869</a>] -         Flink Kubernetes related tests failed to run on aarch64
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-33316'>FLINK-33316</a>] -         Avoid unnecessary heavy getStreamOperatorFactory
+</li>
```

Comment:
As the [Creating+a+Flink+Release](https://cwiki.apache.org/confluence/display/FLINK/Creating+a+Flink+Release) mentioned: `Be appropriately classified as Bug, New Feature, Improvement, etc.`.

I have classified some JIRAs in 1.16.3, may be it's useful for this PR.

https://github.com/apache/flink-web/pull/698

Reply:
Thanks for the information, I updated this PR with selected tickets.

Reply:
Thanks for the update, LGTM